OPS-48 Inital ATMA

motionaviation opened the pull request.

motionaviation committed:
Inital ATMA

motionaviation committed:
Added Basics plus Airspace

dirtyformal committed:
Change structure to /terminal/

motionaviation committed:
Add Deps

motionaviation committed:
Update Dep, Arr and Coord. Added Night STARs

mwr2410 committed:
Minor grammar fixes

motionaviation committed:
Added Arrival Info

motionaviation committed:
Tidy up, Assigned Headings and AR

motionaviation committed:
Grammar/Spelling Updates

dirtyformal commented:
Build is failing due to not having the updated MkDocs file. Will require an edit from me before merged.

dirtyformal committed:
Merge branch 'main' into ops-47

motionaviation committed:
Moved Noise Abatment

dirtyformal committed:
Changes

dirtyformal committed:
Minor Spelling

motionaviation committed:
Minor Updates to Assigned Headings

dirtyformal commented on docs/aerodromes/Class-C/nzaa.md:
```diff
@@ -160,8 +160,8 @@ To ensure a divergent departure occurs due to traffic, AA TMA may request an air
     **JST203**: *"JST203, ready to copy"*  
-    **Auckland Tower**: *"JST203, on departure turn right heading 170, climb four thousand"*
-    **JST203**: *"JST203, on departure turn right heading 170, climb four thousand"*  
+    **Auckland Tower**: *"JST203, on departure turn right heading 170, climb FL360"*
```
I would like to keep them on step-climbs as they're off a lateral-separation path, and separation is not assured.

dirtyformal commented on docs/aerodromes/Class-C/nzaa.md:
```diff
@@ -160,8 +160,8 @@ To ensure a divergent departure occurs due to traffic, AA TMA may request an air
-    **Auckland Tower**: *"JST203, on departure turn right heading 170, climb four thousand"*
-    **JST203**: *"JST203, on departure turn right heading 170, climb four thousand"*  
+    **Auckland Tower**: *"JST203, on departure turn right heading 170, climb FL360"*
+    **JST203**: *"JST203, on departure turn right heading 170, climb FL360"*  
```
Same here.

dirtyformal commented on docs/terminal/atma.md:
```diff
@@ -70,17 +70,17 @@ As track-shortening direct to `TAPAT` or `BOGUN` can interfere with arrivals wit
 
 Assigned Heading departures may be requested by Tower or ATMA in order to allow for more efficient traffic flow from the runway. Controllers can use this method to ensure that a 30° divergence exists between tracks - either keeping the go-around path clear, or by moving an aircraft away from another aircraft's departure path. These shall be issued in accordance with [NZAA - Assigned Headings](../ ...
 
-Due to the early turn required, assigned heading departures shall be given to prop aircraft only.
```
Please leave this in, as this is the case irl. It's sometimes different for Jet vs Jet successives, but it's best if we simplify it.

dirtyformal commented on docs/terminal/atma.md:
```diff
@@ -70,17 +70,17 @@ As track-shortening direct to `TAPAT` or `BOGUN` can interfere with arrivals wit
 !!! example "RTF and coordination for Assigned Heading departure"
-    <span class="coldline">**AA TWR** -> **AA TMA**</span>: "Successive departures. Request ANZ631 assigned heading 190 degrees climbing five thousand then yours for vectors. Second in queue."  
-    <span class="coldline">**AA TMR** -> **AA TWR**</span>: "ANZ631 approved heading 190 degrees climbing five thousand then my vectors. Copy second in line."
+    <span class="coldline">**AA TMA** -> **AA TWR**</span>: "New departure instructions for ANZ631, assigned heading 190 degrees climbing FL360, released"  
```
Please change to a step climb.

dirtyformal commented on docs/terminal/atma.md:
```diff
@@ -70,17 +70,17 @@ As track-shortening direct to `TAPAT` or `BOGUN` can interfere with arrivals wit
-    <span class="coldline">**AA TWR** -> **AA TMA**</span>: "Successive departures. Request ANZ631 assigned heading 190 degrees climbing five thousand then yours for vectors. Second in queue."  
-    <span class="coldline">**AA TMR** -> **AA TWR**</span>: "ANZ631 approved heading 190 degrees climbing five thousand then my vectors. Copy second in line."
+    <span class="coldline">**AA TMA** -> **AA TWR**</span>: "New departure instructions for ANZ631, assigned heading 190 degrees climbing FL360, released"  
+    <span class="coldline">**AA TWR** -> **AA TMA**</span>: "Copy, ANZ631  heading 190 degrees climbing FL360, released"
```
As above.

dirtyformal commented on docs/terminal/atma.md:
```diff
@@ -91,20 +91,18 @@ OCR, RAN or BAY will hand aircraft to ATMA no lower than `A110`, unless co-ordin
 
 !!! hint "RTF for re-joining a STAR"
-    **ATMA**: *New Zealand 677, track direct VINCE to re-join the STAR. When ready descend A050*.
+    **ATMA**: *New Zealand 677, track direct VINCE to re-join the STAR. Descend A050*.
```
As we're directing them to rejoin the STAR, we need to ensure they have radar terrain separation before they hit VINCE and descend by the profile. Should add radar terrain.

dirtyformal reviewed (changes requested):
Please see inline comments.

motionaviation committed:
Changes to Assigned Headings

motionaviation commented:
Have made the above changes and will publish

motionaviation committed:
Edit OCR Noise Abatement Reference

motionaviation committed:
Updated Track Shortening for Deps

motionaviation committed:
Add Table to Track Shortening

motionaviation committed:
Changes to Track Shortening

dirtyformal commented on docs/terminal/atma.md:
```diff
@@ -64,24 +64,56 @@ ATMA should note that all prop-preferential SIDs have a hold-down altitude restr
+- Aircraft to PEBLU, VELMO, GULUT, DUGAN, ELNOS may be tracked direct to these points by TMA
+- Aircraft to MADEP/UPLAR may be tracked to BOGUN/TAPAT respectivly
+
+#### RAN
```
I don't believe the "Flights from" column is necessary, as all of them are ATMA. If there were one or two different ADEPs, then it would make sense. I would also suggest swapping the "may be cleared" and "provided that" columns around as it reads better. IE - qualification then action.

Also, please standardize the amount of full-stops... should be an ellipsis which is three.

motionaviation committed:
Updated Tables

motionaviation committed:
Touch ups

motionaviation committed:
Touch ups

mwr2410 committed:
review of ATMA

- grammar fixes
- update OCR section
- update abbreviations.md
- update OCR/BAY table headings
- add ATIS to abbreviations.md

motionaviation committed:
Uncontrolled IFR Example Clearance

motionaviation committed:
Amend Departures info

mwr2410 committed:
Update .pages

changed Auckland TMA title to Auckland Terminal Area

mwr2410 committed:
Merge branch 'ops-47' of https://github.com/vatnz-dev/sops into ops-47

mwr2410 committed:
Update atma.md

fix uncontrolled IFR table format

dirtyformal committed:
Final Review

mwr2410 committed:
Merge branch 'main' into ops-47
